koku/masu/database/presto_sql/gcp/openshift/reporting_ocpgcptags_summary.sql

```diff
@@ -0,0 +1,102 @@
+WITH cte_tag_value AS (
```

Comment:
I'm not sure this file belongs in the `presto_sql` directory. Thoughts?

I would rather move it to the `sql` directory since this doesn't run on trino.

Reply:
Moved